examples/js/loaders/GLTF2Loader.js

```diff
@@ -1668,10 +1693,12 @@ THREE.GLTF2Loader = ( function () {
 	GLTFParser.prototype.loadMeshes = function () {
 
 		var json = this.json;
+		var extensions = this.extensions;
 
 		return this._withDependencies( [
 
 			"accessors",
+			"bufferViews",
```

Comment:
From my understanding, `bufferView`s will be loaded when `accessor`s are loaded. So the line is not necessary?